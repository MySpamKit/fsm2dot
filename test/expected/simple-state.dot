digraph G {
  node [fontname = "Geneva"]
  edge [fontname = "Geneva"]
  splines=polyline
  none [shape="circle", style="filled", fillcolor=black, label="", width=0.25]
  ready [shape="none", label=<<table style="rounded"><tr><td border="0">ready</td></tr></table>>]
  none -> ready [label=" startup"]
}