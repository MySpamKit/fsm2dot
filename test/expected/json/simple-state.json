{"events":[{"name":"startup","from":"none","to":"ready","label":"startup"}],"choices":[],"states":[{"name":"ready","activities":{"do":[],"entry":[],"entered":[],"exit":[]}}],"final":[],"initial":"none"}